lib/docker2aci.go

```diff
@@ -167,24 +166,28 @@ func SquashLayers(images []acirenderer.Image, aciRegistry acirenderer.ACIRegistr
 	}
 
 	squashedFilename := getSquashedFilename(parsedDockerURL)
-	squashedImagePath := path.Join(outputDir, squashedFilename)
+	squashedImagePath := filepath.Join(outputDir, squashedFilename)
 
-	squashedImageFile, err := os.Create(squashedImagePath)
+	squashedTempFile, err := ioutil.TempFile(outputDir, "squashedFilename-")
```

Comment:
tmpDir can be in another filesystem and the Rename call would fail, I guess we should defer a function that checks if there was any error and removes the temp file :/